releasenotes/notes/initial-ion-trap-support-33686980aa9ec3ae.yaml

```diff
@@ -0,0 +1,17 @@
+---
+features:
+  - |
+    Initial support for executing experiments on ion trap backends has been
+    added.
+    - |
```

Comment:
I think the formatting here is off (and for all the notes below). If the intent here is to make each one a separate feature note the indent level has to be the same (white space matters in yaml). If you want this to be a single note with a list under it you'll need use rst formatting and not yaml formatting.

Reply:
:+1: Moved to separate feature notes in 3c1041beea55fd3f46262179ff3ee295cc33a7bc .